AdyenComponents/BACS Direct Debit/Scenes/Input/BACSDirectDebitInputFormView.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+internal protocol BACSDirectDebitInputFormViewProtocol {
+    func add<T: FormItem>(item: T?)
+    func displayValidation()
+}
+
+internal class BACSDirectDebitInputFormView: FormViewController, BACSDirectDebitInputFormViewProtocol {
+
+    // MARK: - Properties
+
+    internal weak var presenter: BACSDirectDebitPresenterProtocol?
+
+    // MARK: - Initializers
+
+    internal init(title: String,
+                  styleProvider: FormComponentStyle,
+                  localizationParameters: LocalizationParameters? = nil) {
+        super.init(style: styleProvider)
+        self.title = title
+        self.delegate = self
+        self.localizationParameters = localizationParameters
+    }
+
+    // MARK: - BACSDirectDebitInputFormViewProtocol
+
+    internal func add<T: FormItem>(item: T?) {
+        guard let item = item else { return }
+        append(item)
+    }
+
+    internal func displayValidation() {
```

Comment:
There is `showValidation()` in FormViewController that's called in `validate()` already.

That won't work for this one? 2 reasons to ask this:  
- In the continuepayment method, calling `validate()` should ideally be enough without needing `isFormValid` call separately.
- `_ = validate()` is mildly annoying